client/src/components/userProfile/Followers.js

```diff
@@ -0,0 +1,45 @@
+import PaginationTable from '../paginationTable/PaginationTable';
+import followsService from "../../services/follows.service";
+import { useSelector } from "react-redux";
+import { selectUser } from "../../features/auth/auth.selectors";
+
+const Followers = () => {
+
+    const currentUser = useSelector(selectUser);
+
+    const followerColumnDetails = [
+        { field: 'email_from', label: 'Email' },
+    ];
+
+    const loadRequestFn = async (currentPage) => {
+        const response = await followsService.loadFollowers(currentUser.email, currentPage);
+        return {
+            totalElements: response.totalAmount,
+            content: response.data
+          };
+    };
+
+    const removeRequestFn = async(user) => {
```

Comment:
Should you be able to remove those who follow you? It'll ruin their "recommendations"
In my original project they couldn't remove in this section, think about it

Reply:
זה כמו שבאינסטגרם אתה יכול להוריד אנשים שעוקבים אחריך, אני יכולה להוריד את זה

Reply:
קיבלתי